src/components/WalletCard.js

```diff
@@ -58,21 +103,138 @@ const Title = styled.h3`
 
 const Description = styled.p`
   opacity: 0.8;
-  font-size: ${(props) => props.theme.fontSizes.s};
+  font-size: ${({ theme }) => theme.fontSizes.s};
   margin-bottom: 0.5rem;
   line-height: 140%;
 `
 
-const WalletCard = ({ wallet }) => (
-  <Card to={wallet.url} hideArrow={true}>
-    <ImageWrapper background={wallet.brand_color}>
-      <Image fixed={wallet.image.childImageSharp.fixed} alt={wallet.alt} />
-    </ImageWrapper>
-    <Content>
-      <Title>{wallet.name}</Title>
-      <Description>{wallet.description}</Description>
-    </Content>
-  </Card>
-)
+const StyledButtonLink = styled(ButtonLink)`
+  margin: 1rem;
+`
+
+const WalletCard = ({ wallet }) => {
+  const [isCardFlipped, setIsCardFlipped] = useState(false)
+  const duration = 0.3
+  const flipVariants = {
+    shown: {
+      rotateY: 0,
+      transition: {
+        duration,
+      },
+    },
+    frontFlipped: {
+      rotateY: -180,
+      transition: {
+        duration,
+      },
+    },
+    backFlipped: {
+      rotateY: 180,
+      transition: {
+        duration,
+      },
+    },
+  }
+
+  const getFeatureStatus = (feature) =>
+    feature === "TRUE" ? (
+      <Emoji size="0.75rem" text="❌" />
```

Comment:
Do we need to give these aria-labels to represent yes/no?

Reply:
Hm, we haven't in the past for other Emoji usage, though we aren't usually using them to actually convey data. Probably a good idea here